packages/data-grid/src/quickFilter/QuickFilter.scss

```diff
@@ -7,6 +9,19 @@
             padding-right: 8px;
             padding-bottom: 16px;
         }
+        .dg-quick-filter-empty{
+            padding-right: 8px;
```

Comment:
using classname you can give a filter both the .dg-quick-filter-filters and .dg-quick-filter-empty classnames. That means you will not need to repeat the padding-right and padding bottom values.

Reply:
Agree. I must admit I noticed that as well but didn't consider it important. But when you write it down, I can clearly see it makes sense.